test/actions/katello/docker_repository_set_test.rb

```diff
@@ -30,11 +30,13 @@ class DockerTestBase < ActiveSupport::TestCase
     let(:product) { katello_products(:redhat) }
 
     let(:content) do
-      ::Katello::Candlepin::Content.new(id: 'docker-content-123',
-                                        name: 'Docker Content 123',
-                                        type: 'containerimage',
-                                        label: 'content-123',
-                                        contentUrl: content_url)
+      pc = ::Katello::ProductContent::Content.new
+      pc.content_id = 54_321
+      pc.name = 'Docker Content 123'
+      pc.content_type = 'containerimage'
+      pc.label = 'content-123'
+      pc.content_url = content_url
+      pc
```

Comment:
any reason to change this style? (setting each individual value instead of using .new() )

Reply:
Nope, was just getting tests passing. Once we agree on the model structure I'll make factories and clean up the tests.